code/modules/food_and_drinks/recipes/tablecraft/recipes_lizard.dm

```diff
@@ -442,6 +444,7 @@
 	)
 	result = /obj/item/food/steeped_mushrooms
 	category = CAT_LIZARD
+	recipe_flags = RECIPE_KEEP_CONTENTS
```

Comment:
Wrong flag? It's a food item, not a container.

Reply:
My bad, did that one really late into the night and it shows